docs/technical_specs/core/util/unit_costing.rst

```diff
@@ -310,7 +323,11 @@ A centrifugal pump is usually driven by an electric motor, the `self.costing.mot
 
 .. math:: self.motor_purchase_cost = FT * self.costing.motor\_base\_cost * (CE_{index}/500)  (Eq. 22.20)
 
-.. math:: self.costing.motor\_base\_cost = \exp{(5.8259 + 0.13141\log{PC} + 0.053255(\log{PC})^{2} + 0.028628(\log{PC})^{3} - 0.0035549(\log{PC})^{4})}  (Eq. 22.19)
+.. math:: self.costing.motor\_base\_cost = self.costing.motor\_base\_cost\_per\unit * self.costing.number\_of\_units
```

Comment:
Given the size of this, I would suggest we break this section up into sub-pages in future, i.e. a separate page for the equations for each type of model. This might be part of the reworking of costing into using ProcessBlocks.